### ISSUE TITLE
Group Bar Chart 0 value index bar show in middle of the chart. 

### ISSUE THREAD

**Issue by erhspatel** [amcharts/amcharts4#1750]
In Group Bar Chart, when Y axis value is zero, bar stroke is show in middle of the chart. Give suggestion. I have added codepen link.

https://codepen.io/erhspatel/pen/PooZGBE

**Comment by martynasma** [comment]
I suppose the workaround would be to fix minimum value for all your `ValueAxis` to zero. E.g.:

```
{
  "id": "V6",
  "type": "ValueAxis",
  "min": 0,
```

https://codepen.io/team/amcharts/pen/435a93ba012558a9535d219b5326e857?editors=0010

**Comment by erhspatel** [comment]
Thank you :)

**Comment by erhspatel** [comment]
"min": 0 is working fine for positive value but for minus value, It's not working.

**Comment by martynasma** [comment]
What does it mean negative value in context of question about zero values? I'm afraid I don't follow.

**Comment by erhspatel** [comment]
Yes, I have created ticket about zero value base but now, I have issue with negative number in data. It's not showing column proper. I have updated my code pen example.

https://codepen.io/erhspatel/pen/PooZGBE

**Comment by martynasma** [comment]
Please create separate issue for questions/issues that are not related to original topic. This is not a chat or forum, and topics can get quickly out of hand, and hard to follow otherwise.
